test-prow-e2e.sh

```diff
@@ -38,3 +38,6 @@ CHROME_VERSION=$(google-chrome --version) ./test-gui.sh "${1:-e2e}"
 if [ $# -eq 0 ] || [ "$1" == "release" ]; then
   ./test-cypress.sh
 fi
+if [ $# -eq 1 ] && ["$1" == "login" ]; then
+  ./test-cypress.sh 'tests/app/auth-multiuser-login.spec.ts'
+fi
```

Comment:
We can skip running Protractor entirely if `"$1" == "login"`

Reply:
ok, updated `test-prow-e2e.sh`